### DISCUSSION TITLE
some questions about assist2009/ednet datasets.

### DISCUSSION THREAD

**Discussion by AaronKevinAA** [pykt-team/pykt-toolkit#64] [Q&A] [answered]
Hi,recently i run your code to preproce assist2009 and ednet datasets.Then i run wandb_dkt_train.py with preproced datas.
The results show that  assist2009`s auc almost reached about 0.89 and assist2009`s auc almost reached about 0.93.
It is credible?

**Comment by meta-tabchen** [comment, marked as answer]
Thank you for your attention to our project.
For the assist2009 dataset, the test AUC is much higher than our results. We guess the result may be calculated by training datasets, if you provide more details such as logs or command to get the result will be better. We didn't run the ednet dataset now we will run all models on ednet dataset in the future.
